### PR TITLE
PYTHON-2842 Integration tests for observeSensitiveCommands field

### PR DESCRIPTION
**Pull request by prashantmital**
Depends on https://github.com/mongodb/specifications/pull/1043

### CODE REVIEW

**Review by ShaneHarvey — APPROVED** [review]
LGTM

**Comment by prashantmital** [comment]
I'm going to wait for the spec PR to be merged before merging this.

**Pull request merged**
